### PR TITLE
chore: Update gradle version

### LINKED ISSUE #1547: Update the Gradle version
**Describe the chore**
The current version of gradle contains some deprecated code, due to which the build in the PR is failing
For reference, my PR #1539 is failing due to the build error 

*registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)*

and it's solution has been mentioned here, which directly aims to update the gradle version :
https://github.com/google/play-services-plugins/issues/23#issuecomment-448919947
https://github.com/google/play-services-plugins/pull/35

**Would you like to work on the issue?**

- [ ] Yes
- [X] No
- Other:

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1547 

Checklist:

- [ ] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream development branch.

Changes:

Updated gradle version and gradle plugin version.

### CODE REVIEW

**Comment by ShridharGoel** [comment]
Not sure why checkstyle issue is coming in build.

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,9 @@ cache:
       - "$HOME/.gradle/wrapper/"
 install:
   - echo yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-27"
+before_script:
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo "24333f8a63b6825ea9c5514f83c2829b004d1fee" > "$ANDROID_HOME/licenses/android-sdk-license"
```
[the remark this answers is not in the document]
Comment (ShridharGoel) [reply .travis.yml]: Done

[the remark this answers is not in the document]
Comment (ShridharGoel) [reply .travis.yml]: I had added this line to try to fix the license not accepted error.

[the remark this answers is not in the document]
Comment (ShridharGoel) [reply .travis.yml]: Not sure why that error is coming

File: .travis.yml [hunk 2]
```diff
@@ -17,11 +17,10 @@ cache:
       - "$HOME/.gradle/caches/"
       - "$HOME/.gradle/wrapper/"
 install:
-  - echo yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-27"
+  - echo yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-28"
 script:
+  - ./gradlew spotlessCheck
```
Comment (iamareebjamal) [inline .travis.yml]: There's no spotless in orga app

File: .travis.yml [hunk 3]
```diff
@@ -17,11 +17,10 @@ cache:
...
   - ./gradlew build
-  - ./gradlew testPlayStoreDebugUnitTestCoverage
```
Comment (iamareebjamal) [inline .travis.yml]: This is needed

File: .travis.yml [hunk 4]
```diff
@@ -17,11 +17,10 @@ cache:
...
 after_success:
-    - bash <(curl -s https://codecov.io/bash)
```
Comment (iamareebjamal) [inline .travis.yml]: This is needed

**Comment by ShridharGoel** [comment]
@iamareebjamal Please help with the checkstyle error:

```
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:androidCheckstyle'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
        at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
        at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.run(EventFiringTaskExecuter.java:51)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:46)
        at org.gradle.execution.taskgraph.LocalTaskInfoExecutor.execute(LocalTaskInfoExecutor.java:42)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareWorkItemExecutor.execute(DefaultTaskExecutionGraph.java:277)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareWorkItemExecutor.execute(DefaultTaskExecutionGraph.java:262)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:135)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:130)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.execute(DefaultTaskPlanExecutor.java:200)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.executeWithWork(DefaultTaskPlanExecutor.java:191)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.run(DefaultTaskPlanExecutor.java:130)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
Caused by: java.lang.Error: Error was thrown while processing /home/shridhar/AndroidStudioProjects/OpenEventOrgaApp/app/src/androidTest/java/org/fossasia/openevent/app/ExampleInstrumentedTest.java
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:116)
        at org.apache.tools.ant.Task.perform(Task.java:352)
        at org.apache.tools.ant.Task$perform.call(Unknown Source)
        at com.noveogroup.android.check.checkstyle.CheckstyleCheck.performCheck(CheckstyleCheck.groovy:60)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:479)
        at org.gradle.internal.metaobject.BeanDynamicObject.tryInvokeMethod(BeanDynamicObject.java:191)
        at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:78)
        at com.noveogroup.android.check.common.CommonCheck$_apply_closure2$_closure5.doCall(CommonCheck.groovy:84)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:739)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:712)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:131)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:120)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:99)
        ... 31 more
Caused by: java.lang.Error: Error was thrown while processing /home/shridhar/AndroidStudioProjects/OpenEventOrgaApp/app/src/androidTest/java/org/fossasia/openevent/app/ExampleInstrumentedTest.java
        at com.puppycrawl.tools.checkstyle.Checker.processFiles(Checker.java:297)
        at com.puppycrawl.tools.checkstyle.Checker.process(Checker.java:211)
        at com.puppycrawl.tools.checkstyle.ant.CheckstyleAntTask.processFiles(CheckstyleAntTask.java:375)
        at com.puppycrawl.tools.checkstyle.ant.CheckstyleAntTask.realExecute(CheckstyleAntTask.java:335)
        at com.puppycrawl.tools.checkstyle.ant.CheckstyleAntTask.execute(CheckstyleAntTask.java:307)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        ... 54 more
Caused by: java.lang.NoSuchFieldError: WHITESPACE
        at com.puppycrawl.tools.checkstyle.utils.AnnotationUtility.getAnnotation(AnnotationUtility.java:149)
        at com.puppycrawl.tools.checkstyle.utils.AnnotationUtility.containsAnnotation(AnnotationUtility.java:73)
        at com.puppycrawl.tools.checkstyle.checks.naming.MethodNameCheck.visitToken(MethodNameCheck.java:117)
        at com.puppycrawl.tools.checkstyle.TreeWalker.notifyVisit(TreeWalker.java:361)
        at com.puppycrawl.tools.checkstyle.TreeWalker.processIter(TreeWalker.java:498)
        at com.puppycrawl.tools.checkstyle.TreeWalker.walk(TreeWalker.java:303)
        at com.puppycrawl.tools.checkstyle.TreeWalker.processFiltered(TreeWalker.java:178)
        at com.puppycrawl.tools.checkstyle.api.AbstractFileSetCheck.process(AbstractFileSetCheck.java:78)
        at com.puppycrawl.tools.checkstyle.Checker.processFile(Checker.java:314)
        at com.puppycrawl.tools.checkstyle.Checker.processFiles(Checker.java:284)
        ... 62 more
```

**Pull request closed without merging**